LEARNING.md

```diff
@@ -214,16 +229,23 @@ Once you've got the basics of serverless deployment down, you can move on to doi
 
         ```sh
         # Delete the serverless app as `-admin`
-        $ STAGE=sandbox-FIRST-LAST aws-vault exec FIRST.LAST-admin --no-session -- \
+        $ STAGE=sandbox-FIRST-LAST \
+          API_KEY_SECRET=<sandbox key> \
+          aws-vault exec FIRST.LAST-admin --no-session -- \
           yarn lambda:_delete
 
         # Delete Terraform support stack
-        $ STAGE=sandbox-FIRST-LAST aws-vault exec FIRST.LAST --no-session -- \
+        # You will need to manually delete the following before running terraform destroy
+        # 1) db.json inside of the tf-fmd-dogs-sandbox-FIRST-LAST-us-east-1-data (S3) otherwise you will encounter a BucketNotEmpty exception
+        # 2) `FIRST.LAST-admin` inside of the IAM Group `tf-dogs-sandbox-FIRST-LAST-admin` otherwise you will encounter a DeleteConflict exception
+        $ STAGE=sandbox-FIRST-LAST \
```

Comment:
Do we want to perhaps programmatically empty the bucket like done here: https://github.com/FormidableLabs/aws-lambda-serverless-reference/blob/master/package.json#L35-L37 (I forget if our data bucket is versioned -- if not, we can do something slightly different to get the db object).

It not in this PR, might be a good idea to ticket the above out...

Reply:
I'll look into this tomorrow and see if this scripts should be updated or perhaps this dependency get dropped all together with #9